apps/backend/.dockerignore

```diff
@@ -5,4 +5,5 @@
 !db_patches
 !package*
 !tsconfig.json
-!index.ts
\ No newline at end of file
+!index.ts
+src/generated
```

Comment:
Worthy checking if this is working because be are including the whole src directory.